elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -31,94 +31,114 @@ class PfeContentSet extends PFElement {
 
   constructor() {
     super(PfeContentSet, { delayRender: true });
+
+    this._init = this._init.bind(this);
```

Comment:
Perhaps this needs to be in pfelement with a setting like `delayRender: true, mutationObserver: true`?

Reply:
I think it's fine where it is for now. MutationObserver is something we should address post 1.0.